src/apps/dictionaries/pages/ViewDictionaryPage.tsx

```diff
@@ -99,7 +99,7 @@ const ViewDictionaryPage: React.FC<Props> = ({
     >
       <Grid id="viewDictionaryPage" item xs={5} component="div">
         <Paper className="fieldsetParent">
-          <fieldset>
+          <fieldset style={{minWidth: "0"}} >
```

Comment:
Did not understand the usage of this style. We will discuss this.

Reply:
@kirity-mahanty fieldset is the black border outline, this css property keeps the fieldset from overflowing out of page boundary

Reply:
ok.